README.md

```diff
@@ -320,6 +320,10 @@ Note that if a multi-value sub-attribute is `Inheritable`, ancestor classes will
 
 Note: In order to make use of multi-value sub-attributes, the attribute class itself must be marked as "repeatable" as in the example above or PHP will generate an error.  However, that is not sufficient for the Analyzer to parse it as multi-value.  That's because attributes may also be multi-value when implementing scopes, but still only single-value from the Analzyer's point of view.  See the section on Scopes below.
 
+### Finalizing an attribute
+
+Attributes that opt-in to several functional interfaces may not always have an easy time of knowing when to do default handling.  It may not be obvious when the attribute setup is "done."  Attribute classes may therefore opt in to the [`Finalizable`](src/Finalizable.php) interface.  If specified, it is guaranteed to be the last method called on the attribute.  The attribute may then do whatever final preparation is appropriate to consider the object "ready."
```

Comment:
```suggestion
Attributes that opt-in to several functional interfaces may not always have an easy time of knowing when to do default handling.  It may not be obvious when the attribute setup is "done".  Attribute classes may therefore opt-in to the [`Finalizable`](src/Finalizable.php) interface.  If specified, it is guaranteed to be the last method called on the attribute.  The attribute may then do whatever final preparation is appropriate to consider the object "ready".
```

Reply:
Conventional standard English puts the period inside the quite.  My editor for my books yelled at me about that. :smile: I'll correct the opt-in, though.  Thanks.

Reply:
Oh, that's interesting! I'm not a native speaker and didn't know that.